Makefile

```diff
@@ -67,8 +65,8 @@ $(modelGen): target/spec $(openapi-generator-jar)
 	mv $(output)/$(models)/JSON.java $(models)/$@
 
 # Full service + models automation
-bigServices:=balanceplatform checkout payout management legalentitymanagement transfers
-singleFileServices:=balancecontrol binlookup dataprotection storedvalue posterminalmanagement recurring payment capital
+bigServices:=balanceplatform checkout payout management legalentitymanagement
+singleFileServices:=balancecontrol binlookup dataprotection storedvalue posterminalmanagement recurring payment transfers
```

Comment:
Why is transfers in the single file services? does it not have 3 sub parts, transfers, transactions and capital?

Reply:
I did that because it has less than 10 endpoints. I thought it would be a bit much to split it in several services with less than 3 methods each

Reply:
I think we might as well keep to the structure defined in the api explorer. Also could you regenerate the models again @wboereboom? The current ones still use GSON.